client/src/components/NavMenu/NavMenu.css

```diff
@@ -0,0 +1,89 @@
+.nav-container {
+        background-color: #FAE5DC;
+        display: flex;
+        flex-flow: column;
+        justify-content: flex-start;
+        align-content: space-around;
+        align-items: center;
+        height: 100vh;
+        width: 100vw;
+        margin: 0 auto;
+}
```

Comment:
This is some super deep indentation. How is your editor set up? This should be 2 spaces